pkg/v1/features/runs.go

```diff
@@ -22,7 +22,8 @@ type RunsClient interface {
 	GetStatus(ctx context.Context, runId string) (*rest.V1WorkflowRunGetStatusResponse, error)
 
 	// GetDetails retrieves detailed information about a workflow run by its ID.
-	GetDetails(ctx context.Context, runId string) (*rest.WorkflowRunGetShapeResponse, error)
+	// Deprecated: Use Get instead.
+	GetDetails(ctx context.Context, runId string) (*rest.V1WorkflowRunGetResponse, error)
```

Comment:
I assume this is fine since it doesn't work and not many people are using the V1 SDK, but this is a breaking change right?